addons/stock/models/stock_quant.py

```diff
@@ -96,39 +93,36 @@ def _domain_product_id(self):
     tracking = fields.Selection(related='product_id.tracking', readonly=True)
     on_hand = fields.Boolean('On Hand', store=False, search='_search_on_hand')
 
+    # Inventory Fields
+    inventory_quantity = fields.Float(
+        'Counted Quantity', digits='Product Unit of Measure', groups='stock.group_stock_manager',
+        help="The product's counted quantity.")
+    inventory_diff_quantity = fields.Float(
+        'Difference', compute='_compute_inventory_diff_quantity', store=True,
+        help="Indicates the gap between the product's theoretical quantity and its counted quantity.",
+        readonly=True, digits='Product Unit of Measure', groups='stock.group_stock_manager')
+    inventory_date = fields.Date(
```

Comment:
`fields.Datetime()`, in company with high use, it is important to know the time.